.github/workflows/check-yaml.yml

```diff
@@ -0,0 +1,22 @@
+on:
+  pull_request:
+    paths:
+      - '.github/actions/*/*.yml'
+      - '.github/workflows/*.yml'
+      - 'templates/.github/actions/*/*.yml'
+      - 'templates/.github/workflows/*.yml'
```

Comment:
How about `**/actions/**.yml` and `**/workflows/**.yml`? Seems more general and less likely to miss files as we reorganize or add directories.

Reply:
I thought there was no point doing that because we had to be explicit in the action inputs anyway but it turns out that's not entirely true - https://github.com/nwisbeta/validate-yaml-schema/pull/21. Let me adjust it to `**/action.yml`, `**/.github/workflows/*.yml`.